README.md

```diff
@@ -2,11 +2,13 @@
 
 [![Discuss on Matrix at #community:ansible.com](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20on%20Matrix%20at%20%23community:ansible.com&logo=matrix)](https://matrix.to/#/#community:ansible.com)
 
-This repository contains topics that will be discussed and voted by the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
+This repository historically contained topics to be discussed and voted by the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
 
-The **scope** of the topics is changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.
+The **scope** of the topics was changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.
```

Comment:
```suggestion
The **scope** of the topics were changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.
```

Reply:
this is a bit tricky one:)

Reply:
Maybe let's keep is/are, since that's still the scope of the topics - both here and on the forum.

Reply:
I could change it again, but what be correct? Would it be `is` because it's only one scope? Or `are` because of `changes` (plural!)?

I tend to `is` but I'm not a native speaker... maybe someone who is could advise.

Reply:
The scope of the topics `is` changes...

`In` the scope of the topics `are` changes... (rephrased, Changes are in the scope of the topics)

I would say, The scope of the topics `includes` (or `encompasses`) changes...

Reply:
Thanks for the suggestion @cybette!

@Andersson007 @felixfontein Does my [last commit](https://github.com/ansible-community/community-topics/pull/286/commits/df32fc39d86109dbdb0e41cd8a5e0b32042214ee) look OK to you, should we resolve this conversation?

Reply:
It looks good to me!